scripts/taginfo_template.json

```diff
@@ -471,6 +471,62 @@
       "doc_url": "https://openmaptiles.org/schema/#poi",
       "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_p.svg"
     },
+    {
+      "key": "amenity",
+      "value": "bus_station",
+      "object_types": ["node", "area"],
+      "description": "Bus stations are marked by an icon depicting the front of a bus.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_bus.svg"
+    },
+    {
+      "key": "highway",
+      "value": "bus_stop",
+      "object_types": ["node"],
+      "description": "Bus stops are marked by an icon depicting the front of a bus.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_bus.svg"
+    },
+    {
+      "key": "railway",
+      "value": "halt",
+      "object_types": ["node"],
+      "description": "Rail halts are marked by an icon depicting the front of a train.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_rail.svg"
+    },
+    {
+      "key": "railway",
+      "value": "station",
+      "object_types": ["node", "area"],
+      "description": "Rail stations are marked by an icon depicting the front of a train.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_rail.svg"
+    },
+    {
+      "key": "railway",
+      "value": "subway_entrance",
+      "object_types": ["node"],
+      "description": "Subway stations are marked by an icon depicting the front of a train.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_rail.svg"
+    },
+    {
+      "key": "railway",
+      "value": "train_station_entrance",
+      "object_types": ["node"],
+      "description": "Train stations are marked by an icon depicting the front of a train.",
+      "doc_url": "https://openmaptiles.org/schema/#poi",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/icons/poi_rail.svg"
+    },
```

Comment:
It looks like you've added `railway=subway_entrance` and `railway=train_station_entrance` to the taginfo file, but I don't think they're being rendered in the PR?

Reply:
My understanding of the OpenMapTiles schema is that `railway=subway_entrance` translates to to a poi of `class=railway` & `subclass=subway` while `railway=train_station_entrance` translates to a poi of `class=railway` & `subclass=station` but I can double check

Reply:
Upon double checking I think you're right, those two tags end up as `class=entrance` and it's a combination of `railway=station` & `station=subway` that gets translated to `class=railway` & `subclass=subway`

Reply:
Fixed in 759f77b